src/interpreter/kernel/memory.js

```diff
@@ -6,38 +6,41 @@ export const NULL = 0x0;
 // FIXME(sven): remove this ^
 export const ptrsize = 1;
```

Comment:
I don't think this is actually used, I wanted to fake the behavior of low-level pointer but that's maybe unnecessary?

Reply:
That's an interesting point - I guess with the real runtime the store (which the allocator uses) is a proper heap, sorting the funcs, globals etc ... whereas the allocator in this emulator stores references, rather that the data structures themselves. 

I do like the way the emulator closely mimics the real runtime.